test/unit/test_preprocessing.py

```diff
@@ -209,3 +222,28 @@ def test_flatfield_sequences_input():
 
     data = comp.data
     assert data.std() <= 0.01 * np.mean(seqs[1][0].data)
+    
+def test_stitching():
+    image_received = _prepare_stitching_test_img()
```

Comment:
Please do not capture multiple returns in a single variable, since this makes the code less readable. Please capture all variables separately:

```
img0, img1, img2, img3, image = _prepare_stitching_test_img()
```